conanfile.py

```diff
@@ -44,27 +41,42 @@ def requirements(self):
             self.requires("gtest/1.14.0")
             self.requires("boost/1.84.0")
         if self.options.build_unbundled: #each componenet is built independently 
-            self.requires("up-cpp/0.1.1-dev")
-            if self.options.zenoh_package:
-                self.requires("zenohc/cci.20240213")
-        if self.options.build_cross_compiling :
+            self.requires("up-cpp/x.y.z")
             self.requires("zenohc/cci.20240213")
 
     def generate(self):
         tc = CMakeToolchain(self)
         tc.variables["BUILD_TESTING"] = self.options.build_testing
-        tc.variables["CROSS_COMPILE"] = self.options.build_cross_compiling
         tc.generate()
 
     def build(self):
+        if os.environ.get("BUILD_DIR_FULL_PATH") is not None:
+            return
```

Comment:
Please provide documentation of the environment and config variables being added to the build system. Any new developer of packager should be able to find out what this does and how to make use of it.

Reply:
BUILD_DIR_FULL_PATH enables to wrap local build in conan package